driver/web/rest/apis.go

```diff
@@ -499,57 +513,75 @@ type getUserGroupsResponse struct {
 // @Tags Client-V1
 // @Accept  json
 // @Param APP header string true "APP"
+// @Param title query string false "Deprecated - instead use request body filter! Filtering by group's title (case-insensitive)"
+// @Param category query string false "Deprecated - instead use request body filter! category - filter by category"
+// @Param privacy query string false "Deprecated - instead use request body filter! privacy - filter by privacy"
+// @Param offset query string false "Deprecated - instead use request body filter! offset - skip number of records"
+// @Param limit query string false "Deprecated - instead use request body filter! limit - limit the result"
+// @Param include_hidden query string false "Deprecated - instead use request body filter! include_hidden - Includes hidden groups if a search by title is performed. Possible value is true. Default false."
+// @Param data body model.GroupsFilter true "body data"
 // @Success 200 {array} getUserGroupsResponse
 // @Security AppUserAuth
 // @Security APIKeyAuth
 // @Router /api/user/groups [get]
 func (h *ApisHandler) GetUserGroups(clientID string, current *model.User, w http.ResponseWriter, r *http.Request) {
 
-	var category *string
+	var groupsFilter model.GroupsFilter
+
 	catogies, ok := r.URL.Query()["category"]
 	if ok && len(catogies[0]) > 0 {
-		category = &catogies[0]
+		groupsFilter.Category = &catogies[0]
 	}
 
-	var privacy *string
 	privacyParam, ok := r.URL.Query()["privacy"]
 	if ok && len(privacyParam[0]) > 0 {
-		privacy = &privacyParam[0]
+		groupsFilter.Privacy = &privacyParam[0]
 	}
 
-	var title *string
 	titles, ok := r.URL.Query()["title"]
 	if ok && len(titles[0]) > 0 {
-		title = &titles[0]
+		groupsFilter.Title = &titles[0]
 	}
 
-	var offset *int64
 	offsets, ok := r.URL.Query()["offset"]
 	if ok && len(offsets[0]) > 0 {
 		val, err := strconv.ParseInt(offsets[0], 0, 64)
 		if err == nil {
-			offset = &val
+			groupsFilter.Offset = &val
 		}
 	}
 
-	var limit *int64
 	limits, ok := r.URL.Query()["limit"]
 	if ok && len(limits[0]) > 0 {
 		val, err := strconv.ParseInt(limits[0], 0, 64)
 		if err == nil {
-			limit = &val
+			groupsFilter.Limit = &val
 		}
 	}
 
-	var order *string
 	orders, ok := r.URL.Query()["order"]
 	if ok && len(orders[0]) > 0 {
-		order = &orders[0]
+		groupsFilter.Order = &orders[0]
 	}
 
-	groups, err := h.app.Services.GetUserGroups(clientID, current, category, privacy, title, offset, limit, order)
+	requestData, err := io.ReadAll(r.Body)
```

Comment:
Thanks for raising this @shurwit ! I'm aware, but we already have released endpoints which don't produce issues so far. Of course this is not 100% true in case of browsers or clouds like GCP.  I would prefer to address them with v3 endpoints in another ticket due to the involved client release as well

Reply:
Actually this would be addressedin in this PR.

Reply:
Thanks @mdryankov, I know it does work as expected in our environment but I still think it's probably best to avoid it since other developers may want to use our APIs in other environments in the future.